detect_secrets/pre_commit_hook.py

```diff
@@ -19,13 +19,13 @@
 log = get_logger(format_string='%(message)s')
 
 
-def parse_args(argv):
+def parse_args(argv):  # pragma: no cover (Mocked)
     return ParserBuilder()\
         .add_pre_commit_arguments()\
         .parse_args(argv)
 
 
-def main(argv=None):
+def main(argv=sys.argv[1:]):
```

Comment:
Must have missed this last Friday, so by default `argparse` looks at `sys.argv`, if what you pass to it when calling `parse_args()` is None, since I wanted to check both provided arguments, and sys.argv, I explicitly made `argv` come from `sys.argv` when nothing is provided.

Reply:
Makes sense.